### ISSUE TITLE
Galleon not available on mac app store

### ISSUE THREAD

**Issue by utdrmac** [Cryptonomic/Deployments#47]
Says here (https://github.com/Cryptonomic/Deployments/wiki/Galleon:-Releases), quite explicitly:

> It's always better to get Galleon from an app store, especially for automatic updates!

1. Open Mac App store.
2. Search 'galleon'
3. No Results Found.

**Comment by gologo13** [comment]
I had the same issue.

**Comment by shaun-19826** [comment]
You can access the galleon wallet via the galleon web wallet though. I can give you the weblink if needed

**Comment by utdrmac** [comment]
@shaun-19826 Yes, I know that. This issue is that the instructions to download Galleon SPECIFICALLY tell you to go to the mac app store to get it and it's not there.
